app/javascript/shared/components/tags.jsx

```diff
@@ -300,18 +300,33 @@ export class Tags extends Component {
       input.value.slice(range[1], input.value.length);
     /* eslint-disable-next-line react/destructuring-assignment */
     this.props.onInput(newInput);
-    this.resetSearchResults();
+    this.fetchTopTagSuggestions();
     this.clearSelectedSearchResult();
   }
 
+  fetchTopTagSuggestions() {
+    const { topTags = [] } = this.state;
+    if (topTags.length > 0) {
```

Comment:
the top tags are cached while we're on the page, since it seems unnecessary to fetch them every time